lib/api/controllers/authController.js

```diff
@@ -53,20 +77,18 @@ class AuthController {
     return this.kuzzle.passport.authenticate({query: request.input.body, original: request}, strategy)
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Split this 102 characters long line (which is greater than 80 authorized). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ALineLength)